app/models/post.rb

```diff
@@ -4,8 +4,8 @@ class Post < ApplicationRecord
   has_many :likes, foreign_key: :post_id
 
   validates :title, presence: true, length: { minimum: 3, maximum: 250 }
-  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  # validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  # validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] **[OPTIONAL]** Kindly add these validations as they were in a previous project instead of commenting them out. Validations are important for securing our apps.